pkg/controller/perconaservermongodb/smart.go

```diff
@@ -152,8 +152,9 @@ func (r *ReconcilePerconaServerMongoDB) smartUpdate(cr *api.PerconaServerMongoDB
 		}
 	}
 
-	log.Info("doing step down...")
-	err = mongo.StepDown(context.TODO(), client)
+	unsafe := cr.Spec.UnsafeConf && len(list.Items) == 1
```

Comment:
Let's call this var `force`

Reply:
Btw, do we need to check `cr.Spec.UnsafeConf` ?

Reply:
or even `forceStepDown`

Reply:
also, what if we have replica set size = 3 in CR, but only one member in the list?

Reply:
should we check the replset size to decide if force required? we won't need the check `r.Spec.UnsafeConf` this way?

Reply:
yes, please check replset size

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/651/commits/bc2a469c7302dfa68d26504037062857caed8f9b